section4/exercises/classes_and_objects_1.rb

```diff
@@ -0,0 +1,45 @@
+class MyCar
+  attr_accessor :color
+  attr_reader :year
+
+  def initialize(y, c, m)
```

Comment:
This is great, but be cautious using non-descriptive variables for future cases :)